### DISCUSSION TITLE
Align log output color with theme?

### DISCUSSION THREAD

**Discussion by pat-s** [woodpecker-ci/woodpecker#2228] [Request a Feature]
To look something like

![image](https://github.com/woodpecker-ci/woodpecker/assets/8430564/cac109cd-0991-495d-800d-a80bf4d671b1)

Instead of the red ansi-based output color which is currently used. I always think of an error when I see red - but here red is just meant to show code which is being executed.

Opinions?

**Comment by lafriks** [comment]
I don't quite understand the proposal. It displays colors as they would be in console output, I don't think we should change them or it's about something else?

**Reply by pat-s** [reply]
Currently the main color for "what is being executed" text is `ansi-bright-red`. I'd like to change that. Ideally this would be configurable on the user side but this might be more complicated.

The screenshot above is adjusted to green and matches the other green elements.

**Reply by lafriks** [reply]
doesn't tool itself (docker build) does that not woodpecker runner?

**Reply by pat-s** [reply]
Sorry I don't understand what you mean. Can you rephrase?

**Reply by lafriks** [reply]
I mean that I don't see where woodpecker would be adding that color anywhere

**Reply by pat-s** [reply]
It's done by `ansi-to-html` and I think you did the PR :) I don't fully understand the former, i.e. how the color is determined and where the ANSI codes come from. 

I've browsed the code and couldn't backtrace the CSS class. Any pointer welcome...

**Reply by 6543** [reply]
ansi should come from the log stream ... as it just capture the raw "terminal" output witch also can have collor coding infos in it

**Comment by qwerty287** [comment]
As explained above there's no "theming" on logs, so I guess we can close this?
